wagtailio/static/sass/components/_quotes.scss

```diff
@@ -0,0 +1,72 @@
+.quotes {
+    &__header {
+        color: $color--off-black;
+        padding-bottom: 30px;
+        grid-column: 2 / span 4;
```

Comment:
Suggestion: Change this to `grid-column: 2 / span 2` as we only have [2 columns on mobile](https://github.com/wagtail/wagtail.org/blob/next/wagtailio/static/sass/components/_grid.scss#L3) that we want to fill : )